azext_iot/tests/digitaltwins/test_dt_resource_lifecycle_int.py

```diff
@@ -25,8 +25,7 @@
     EP_SERVICEBUS_TOPIC,
     MOCK_RESOURCE_TAGS,
     MOCK_RESOURCE_TAGS_DICT,
-    MOCK_DEAD_LETTER_SECRET,
-    MOCK_DEAD_LETTER_ENDPOINT,
+    MOCK_DEAD_LETTER_PLACEHOLDER,
```

Comment:
What is the purpose for MOCK_DEAD_LETTER_PLACEHOLDER

Reply:
good q - it seems like the deadletter secret isn't checked either way (says it is flaky) so will just remove this for now... later we can try adding checks for the deadletter secret